index.css

```diff
@@ -370,9 +567,54 @@ footer {
 }
 
 .underline {
+  display: flex;
   margin: 23px auto 8px auto;
   height: 5px;
   width: 134px;
   background: #0d151e;
   border-radius: 10px;
 }
+
+.name-control {
+  display: none;
+  grid-template-columns: 1fr 1fr;
+  gap: 20px;
+}
+
+.full-name {
+  display: flex;
+}
+
+@media screen and (min-width: 768px) {
+  .contact-info {
+    text-align: left;
+    margin-top: 3.5rem;
+  }
+
+  .contact-section {
+    min-height: 614px;
+  }
```

Comment:
- [ ] Just like the projects section, your contact section has a different background color from the template. Kindly add a white background color to match with the template design.

Reply:
![microverse4](https://user-images.githubusercontent.com/95437850/206191728-2c1f9013-75e5-4ef4-850c-03763b0ec7f0.png)
my css code :`.section-container {
  background: #e5e5e5;
  min-height: 100vh;
  display: grid;
  align-items: center;
  justify-content: center;
}` also correct as per the design